app/views/admin/unattached_promo_registrations/index.html.slim

```diff
@@ -1,110 +1,113 @@
         .panel--header = "Create campaigns"
         hr
         .
           = text_field_tag "campaign_name", nil, placeholder: "Campaign name"
-          = submit_tag "+ Campaign", class: "btn btn-info"
+          = submit_tag "Create", class: "btn btn-info", style: "float: right"
   .row
-    .panel.panel--single
-      .panel--header = "Manage"
-      span style="color: darkgrey; font-size: 14px;" = "Stats last refreshed #{distance_of_time_in_words (Time.now - Rails.cache.fetch('unattached_promo_registration_stats_last_synced_at'))} ago. Stats in reports are always up-to-date."
-      hr
-      = form_tag admin_unattached_promo_registrations_path, method: :get, id: "filter-by-campaign"do
-        = select_tag :filter, options_for_select(@campaigns + ["Not assigned", "All codes"], @current_campaign)
-        = submit_tag "Filter by campaign", class: "btn btn-success"
-      = form_tag admin_unattached_promo_registrations_path, method: :patch, id: "unattached-referral-code-form" do
-        table.table
-          tr
-            th 
-            th = "Code"
-            th data-toggle="tooltip" data-placement="top" title="Tooltip"= "Campaign"
-            th
-              = "Status"
-              span.tf-tooltip
-                span.icon= render "icon_help"
-                span.tf-tooltip-content
-                  span.tf-tooltip-content-heading= "Status"
-                  span.tf-tooltip-content-content== "A code's stats will not be tracked or updated while it's status is 'paused.'"
-            th = "Downloads"
-            th 
-              = "Installs"
-              span.tf-tooltip
-                span.icon= render "icon_help"
-                span.tf-tooltip-content
-                  span.tf-tooltip-content-heading= "Installs"
-                  span.tf-tooltip-content-content== "An install is counted when a user downloads the browser and opens it the first time."
+  .panel.panel--white.panel--table
+    .panel--header = "Manage"
+    . style="color: darkgrey; font-size: 14px; margin-bottom: 5px;"
+      - if Rails.cache.fetch('unattached_promo_registration_stats_last_synced_at').nil?
+        = "Stats have not been refreshed."
+      - else 
+        = "Stats last refreshed #{distance_of_time_in_words(Time.now - Rails.cache.fetch('unattached_promo_registration_stats_last_synced_at'))} ago. Stats in reports are always up-to-date."
+    = form_tag admin_unattached_promo_registrations_path, method: :get, id: "filter-by-campaign" do
+      = select_tag :filter, options_for_select(@campaigns + ["Not assigned", "All codes"], @current_campaign)
+      = submit_tag "Filter by campaign", class: "btn btn-success"
+    = form_tag admin_unattached_promo_registrations_path, method: :patch, id: "unattached-referral-code-form" do
+      table.table
+        tr
+          th 
+          th = "Code"
+          th data-toggle="tooltip" data-placement="top" title="Tooltip"= "Campaign"
```

Comment:
What does `title="Tooltip"= "Campaign"` do?

Reply:
leftover from an attempt to use bootstrap tooltips. can remove